PowerEditor/installer/nsisInclude/langs4Installer.nsh

```diff
@@ -15,11 +15,12 @@
 ; along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 
-; Set languages (first is default language)
-;!insertmacro MUI_LANGUAGE "English"
+!define MUI_LANGDLL_ALWAYSSHOW
 !define MUI_LANGDLL_ALLLANGUAGES
 ;Languages
 
+  ; Set the Notepad++ static languages list
+  !insertmacro MUI_LANGUAGE "English" ; first is the default language (important for a possible language preselection failure...)
```

Comment:
Have this code has been tested?
My binary which works for you is the following code:
```nsis
; Set languages (first is default language)
!insertmacro MUI_LANGUAGE "English"
!define MUI_LANGDLL_ALLLANGUAGES
;Languages

  !insertmacro MUI_LANGUAGE "Afrikaans"
  !insertmacro MUI_LANGUAGE "Albanian"
  !insertmacro MUI_LANGUAGE "Arabic"
  ;!insertmacro MUI_LANGUAGE "Armenian"
  !insertmacro MUI_LANGUAGE "Basque"
  !insertmacro MUI_LANGUAGE "Belarusian"
  !insertmacro MUI_LANGUAGE "Bosnian"
  !insertmacro MUI_LANGUAGE "Breton"
  !insertmacro MUI_LANGUAGE "Bulgarian"
  !insertmacro MUI_LANGUAGE "Catalan"
  !insertmacro MUI_LANGUAGE "Corsican"
  !insertmacro MUI_LANGUAGE "Croatian"
  !insertmacro MUI_LANGUAGE "Czech"
  !insertmacro MUI_LANGUAGE "Danish"
  !insertmacro MUI_LANGUAGE "Dutch"
  ;!insertmacro MUI_LANGUAGE "English"
  ...
```

Reply:
Yes, of course, many times by me (I use a Win11 VM with the Icelandic locale for that) and by @MalakymR (he can test on a computer with that more serious NSIS problem - skipping the lang-selection dlg + "Install Language" regkey...).